lib/web-service/nginx-util.ts

```diff
@@ -88,6 +115,41 @@ function getSameSiteCookiesConfig(): string {
   return fs.readFileSync(resolvePath("nginx/samesite.conf"), "utf8");
 }
 
+function getPartitionedCookiesConfig(locations?: string[]): string {
+  if (!locations) {
+    return "";
+  }
+
+  return locations
+    .map((location) =>
+      createProxyRouteConfig(location, `http://application${location}`, [
```

Comment:
Hmm I think proxy only needs the base URL, location will be appended when it's called.